common/scaffolding/typographie.html

```diff
@@ -0,0 +1,45 @@
+---
+{
+	"title": "Typographie",
+	"language": "fr",
+	"altLangPage": "typography.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
```

Comment:
Remove the breadcrumb as it is there by default now.

```suggestion
```

Reply:
Fixed and addressed in commit "Fixed links and removed breadcrumbs".